src/hotspot/share/gc/g1/g1EvacFailure.cpp

```diff
@@ -133,7 +133,6 @@ class RemoveSelfForwardPtrObjClosure {
 #endif
       }
     }
-    _cm->clear_range_in_prev_bitmap(mr);
```

Comment:
The comment in line 107 needs to be updated to something like:

```
// Fill the memory area from start to end with filler objects, and update the BOT
// accordingly. Since we clear and use the prev bitmap for marking objects that
// failed evacuation, there is no work to be done there.
```